recipes/vrs/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "vrs" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/facebookresearch/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: cd0f2a043a8ee2386c4bec61987e9c25f8de748f4d63bbe34a0d15115d858a98
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - ninja  # [not win]
+  host:
+    - libboost-devel
+    - fmt
+    - libjpeg-turbo
+    - lz4
+    - portaudio
+    - xxhash
+    - zlib
+    - qt
+    - qt-main
+    - mesalib  # [linux]
+    # dependencies to be added for RUN step
+    - libpng
+    - lz4-c
+    - zstd
+    - libboost
+
+test:
+  commands:
+    - test -f $PREFIX/bin/vrsplayer  # [not win]
```

Comment:
Can you please add tests for Windows, too?

Reply:
Do you know if I can use the same command line to know if the file is present?
I.e. is the default path different for windows?

Reply:
Probably the following is what you want:
`if not exist %LIBRARY_PREFIX%\bin\vrsplayer.exe exit 1`

Reply:
@xhochy Thank you, your suggestion worked.